### DISCUSSION TITLE
Why the line of code for folder is shown as a negatice number?

### DISCUSSION THREAD

**Discussion by qintian-qe** [oracle/opengrok#4694] [General]
**Describe the bug**
A clear and concise description of what the bug is. Make sure you have done a search of the existing issues to prevent creating duplicates.

If you provide the exact version of OpenGrok, JDK used, OS(and its version) used and Tomcat(or your webapp server) used it can help figuring out an environment issue.
For performance problem OS and JDK tunables might be needed. For SCM problems also version of SCM is helpfull (e.g. some mercurial versions have issues, some not).

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by qintian-qe** [comment]
![Uploading 企业微信截图_1732778464803.png…]()

**Comment by vladak** [comment]
What OpenGrok version are you running ?

**Reply by qintian-qe** [reply]
version 1.7.10
